javaparser-symbol-solver-testing/src/test/test_sourcecode/javasymbolsolver_0_6_0/src/java-symbol-solver-logic/com/github/javaparser/symbolsolver/logic/InferenceVariableType.java

```diff
@@ -95,7 +95,7 @@ public String describe() {
 
     @Override
     public boolean isAssignableBy(Type other) {
-        throw new UnsupportedOperationException();
+        throw new UnsupportedOperationException(String.format("Unable to assign the type %s", other));
```

Comment:
Double check this please -- "Unable to assign..." suggests that we have tried to do an assignment (not the case, here we've just done a check to see if it could be possible).

Could it be that this is one case where this should never be possible, as opposed to being not yet implemented? (note: I don't know the answer!)

Possibly worth taking a look at the documentation:
https://docs.oracle.com/javase/8/docs/api/java/lang/Class.html#isAssignableFrom-java.lang.Class-